go/kubebuilder/scaffold_go_operator.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
```

Comment:
Could we add the script in `.generate/` as well?
Also, wdyt about its name be `gen-go-sample.sh `?